native/src/boot/bootimg.cpp

```diff
@@ -699,27 +739,40 @@ void repack(const char *src_img, const char *out_img, bool skip_comp) {
 
     // ramdisk
     off.ramdisk = lseek(fd, 0, SEEK_CUR);
+    if (!hdr->is_vendor() && hdr->header_version() == 4 && boot.ramdisk_table_entries.empty() && access(RAMDISK_FILE, R_OK) == 0) {
+        // Add a ramdisk table entry for v4 GKI boot images with an initially empty ramdisk section
+        new_ramdisk_table_entries.emplace_back(make_unique<vendor_ramdisk_table_entry_v4>(), LZ4_LEGACY);
```

Comment:
where is the initialization of the new `vendor_ramdisk_table_entry_v4`?

Reply:
section was removed